cves/CVE-2017-14170.yml

```diff
@@ -90,14 +102,22 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 900f39692ca0337a98a7cf047e4e2611071810c2
-  note: ''
-- commit: ''
-  note: ''
+  note: 'Official fix commit'
+- commit: 'c01f799314c3254a98c415ccf99acd501bdbd9f2'
+  note: 'Cherry pick of fix commit for version 3.3.4'
+- commit: '9cbac3602610afa0867b03bc1475c5c13441d096'
+  note: 'Cherry pick of fix commit for version 3.3.4'
```

Comment:
Are these cherry-picked commits supposed to be for the same version?

Reply:
Nope, I just overwrote the same one twice when filling it in after copy-pasting